content/docs/context.md

```diff
@@ -9,20 +9,20 @@ Context, prop'ları her seviyede manuel olarak geçmek zorunda kalmadan bileşen
 Tipik bir React uygulamasında veri prop'lar aracılığıyla yukarıdan aşağıya aktarılır (üst bileşenlerden alt bileşenlere), fakat bu bir uygulamada birçok bileşene ihtiyaç duyulan belirli tipteki prop'lar (örneğin; lokalizasyon, arayüz teması) için kullanışsız olabilir. Context, bileşen ağacın her bir seviyesi üzerinden açıkça bir prop geçirmeden, bileşenler arasında bu gibi değerleri paylaşmanın bir yolunu sağlar.
 
 
-- [Context Ne Zaman Kullanılır {#when-to-use-context}](#context-ne-zaman-kullan%c4%b1l%c4%b1r-when-to-use-context)
```

Comment:
`[Context Ne Zaman Kullanılır](#when-to-use-context)`

Reply:
I did. Hope, did not miss any.
Also links may be broken because of unnecessary marks (``) removed them also.